pkg/virt-launcher/virtwrap/device/sriov/hostdev.go

```diff
@@ -32,6 +32,8 @@ type pool interface {
 	Pop(key string) (value string, err error)
 }
 
+const AliasPrefix = "sriov-"
```

Comment:
The `sriov` prefix seems to me redundant. The network name (`iface.Name`) should be enough the determine it is an sriov interface.

Reply:
The prefix is the the signature used to mark if it is an SR-IOV device or not. The network name is a nice-to-have addition.
If we will just keep the network name, there will be a need to lookup every time and see if it correlates to a SR-IOV network.

In its usage, no such lookup is needed.